src/lib/index.d.ts

```diff
@@ -1,8 +0,0 @@
-#!/usr/bin/env node
-
-declare module '*'
```

Comment:
Yea, this is something I really did not like also if I'm correct it had `any` type assigned, the whole point of typescript is to niot use any, unless there are no other ways or an edge case